index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
```

Comment:
- [ ] [OPTIONAL] For your next projects, please, do not use styling direct for HTML tags, give class name for styling.